packages/botonic-react/src/util/dom.js

```diff
@@ -26,9 +29,17 @@ export const scrollToBottom = ({ timeout = 200, behavior = 'smooth' } = {}) => {
   }
 }
 
-export const getWebchatElement = () =>
-  document.getElementById('botonic-webchat')
+export const getWebchatElement = host =>
+  host && host.querySelector('#botonic-webchat')
 
-export const setWebchatElementHeight = newHeight => {
-  getWebchatElement().style.height = newHeight
+// https://stackoverflow.com/questions/9457891/how-to-detect-if-domcontentloaded-was-fired
+export const onDOMLoaded = f => {
+  if (/complete|interactive|loaded/.test(document.readyState)) {
+    // In case the document has finished parsing, document's readyState will
+    // be one of "complete", "interactive" or (non-standard) "loaded".
+    f()
+  } else {
+    // The document is not ready yet, so wait for the DOMContentLoaded event
+    document.addEventListener('DOMContentLoaded', f, false)
```

Comment:
give an explicit name to f? if refers to fire()?

Reply:
f = function, but yeah, maybe I'll put a more descriptive name